### PR TITLE
Make post reviews route

### PR DESCRIPTION
**Pull request by milofultz**

### CODE REVIEW

File: __tests__/queries.test.js [hunk 1]
```diff
@@ -154,6 +177,92 @@ describe("Query Tests", () => {
...
+
+    after((done) => {
+      Review.deleteMany({})
+        .then(() => CharacteristicReview.deleteMany({}))
+        .then(() => Characteristic.deleteMany({}))
+        .then(() => done());
+    });
+  });
+
+  describe('postReview', () => {
```
Comment (NylonEric) [inline __tests__/queries.test.js:189]: You can use a before() hook if you are manipulating the same data and don't want to re-initialize before each test.

File: server/db/queries.js [hunk 2]
```diff
@@ -9,6 +9,12 @@ const {
 
 // Helper functions
 
+const STATUS = {
```
Comment (NylonEric) [inline server/db/queries.js:12]: Clever way of selecting status codes!

**Review by NylonEric — APPROVED** [review]
LGTM
* I made one comment about building your db once before a block of tests as an option.

File: server/db/queries.js [hunk 3]
```diff
@@ -20,6 +26,8 @@ const getNextValue = async (collectionName) => {
   return response.value
 };
 
+const getCurrentUnixTimestamp = () => Math.floor(new Date().getTime() / 1000);
```
[the remark this answers is not in the document]
Comment (milofultz) [reply server/db/queries.js:29]: Word. I'll remember this for when I have to optimize.

File: __tests__/queries.test.js (hunk 1 above)
Comment (milofultz) [reply __tests__/queries.test.js:189]: I'm not sure if I need to isolate them as I am, but I figure starting from the same entry point every time is safe? I dunno, I guess I'll find out

Comment (milofultz) [reply __tests__/queries.test.js:189]: I'm not sure if I need to isolate them as I am, but I figure starting from the same entry point every time is safe? I dunno, I guess I'll find out

**Pull request merged**